tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -103,6 +103,10 @@ def prepare_config_and_inputs(
     ):
         input_ids = ids_tensor([self.batch_size, self.seq_length], self.vocab_size)
 
+        # Only left padding is valid
+        attention_mask = torch.ones(size=(self.batch_size, self.seq_length), device=input_ids.device, dtype=torch.long)
+        attention_mask[0, :1] = 0
```

Comment:
Added a mask, maybe for some other tests as well.

Reply:
Alright, is it intended that it is only tuned out for the first element of the batch?